### PR TITLE
{bio}[GCC/10.2.0] NGS 2.10.9

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'NGS'
+version = '2.10.9'
+
+homepage = 'https://github.com/ncbi/ngs'
+description = """NGS is a new, domain-specific API for accessing reads, alignments and pileups
+produced from Next Generation Sequencing."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
```
Comment (Micket) [inline easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCC-10.2.0.eb:10]: Do we want to move this into GCC instead of GCCcore?

Comment (fizwit) [reply easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCC-10.2.0.eb]: No, sorry.  Moving back to GCCcore.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/c7c0de4fcc5f4e39ee02abe3e840dab7 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node371.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b2640f5342b1c1e7340617665ff0843f for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @fizwit !

**Pull request merged**